src/lib/contract-client/index.ts

```diff
@@ -75,6 +75,11 @@ class ContractClient {
     return circulatingSupply;
   };
 
+  getDistributedToday = async (): Promise<string> => {
```

Comment:
Just `string` here is enough I believe

Reply:
`Promise<string>` is correct here since it's an async function